Carbon.ScheduledTasks/prism.lock.json

```diff
@@ -0,0 +1,19 @@
+{
+  "PSModules": [
+    {
+      "name": "Carbon.Accounts",
+      "version": "1.0.0-rc5",

```

Comment:
Lock RCs?

Reply:
Perhaps if there is a way to prism.lock like 'use rc only if this is rc' - although how to get that inversion 🩰